site-docs/src/main/paradox/dynamodb/usage/index.md

```diff
@@ -0,0 +1,40 @@
+# DynamoDB Client
+
+The client constructor takes a `DynamoDbAsyncClient` object. The default `apply` method passes one in with sensible defaults, but you can pass your own in if necessary.
+
+```scala
+val dynamoDbAsyncClient: DynamoDbAsyncClient = ???
+val clientWithCustom = new DADynamoDBClient(dynamoDbAsyncClient)
```

Comment:
Do you want to use the apply method in the example? So you can get rid of `val dynamoDbAsyncClient: DynamoDbAsyncClient` and do `val clientWithCustom = DADynamoDBClient()` That's what most calling clients will do I reckon. What do you think?

Reply:
Do you mean like what is on line 9? I copied what you did in the S3 client's `index.md` file. I assumed you did what you did in order to show two ways that a user could instantiate the client: custom (via the line 6 and 7)  or default (line 9); did I misinterpret that?

Reply:
Sorry, I'm blind, I didn't see line 9. Ignore me.